website/docs/components/alert/index.md

```diff
@@ -21,6 +21,10 @@ navigation:
   @include "partials/code/component-api.md"
 </section>
 
+<section data-tab="Content">
+  @include "partials/content/content.md"
+</section>
+
```

Comment:
@jorytindall Do any of our other components include "Content" tabs?

Reply:
The [Table](https://helios.hashicorp.design/components/table?tab=content) component does as there was just a bunch of new guidelines and recommendations added that blew up the guidelines.

Reply:
It seems to be causing a test failure: https://github.com/hashicorp/design-system/actions/runs/5906146301/job/16021760000?pr=1576#step:5:43

Reply:
Thanks for fixing! @jorytindall